app/containers/AskQuestion/validate.js

```diff
@@ -0,0 +1,18 @@
+import { getActivePopover } from 'utils/popover';
+import messages from './messages';
+
+export const postQuestionValidator = (
+  profileInfo,
+  postButtonId,
+  translations,
+) => {
+  const minRatingToAskQuestion = 0;
+
+  if (profileInfo.rating < minRatingToAskQuestion) {
+    const message = `${
+      translations[messages.notEnoughRating.id]
+    } ${minRatingToAskQuestion}`;
+    getActivePopover(postButtonId, message);
+    throw new Error(message);
```

Comment:
I see multiple places where exception is raised when validation fails. Does that show error in console? I do not think we need to raise error. This is normal scenario. Exception should be raised only if there is real error in the logic.